tiles-generation/cdk/README.md

```diff
@@ -0,0 +1,59 @@
+# Tiles generation pipeline with CDK
+
+ This project in `tilesGenerationPipeline` folder helps to build a raster map tiles generaion pipeline on Amazon ECS in AWS CDK. The docker base image is come from https://github.com/Overv/openstreetmap-tile-server.
+
+## Diagram
+![alt text](./tiles-generation-diagram.png).
```

Comment:
Typo in cluster. Also, diagram inside Cloudformation is a little bit confusing. I would maybe break the diagram into numbered steps.

Reply:
Updated, I added steps explanation for the diagram.